index.html

```diff
@@ -27,7 +26,6 @@
 			</label>
 			<button>TINY BUTTON!!</button>
 		</form>
-
 		<script type="module" src="./home/home.js"></script>
```

Comment:
everything happening in home.js should be happening in app.js in the parent folder. It is very traditional to have index.html and app.js work together for the home page.